apps/studio/src/features/editing-experience/components/RootStateDrawer.tsx

```diff
@@ -269,7 +273,19 @@ export default function RootStateDrawer() {
               </VStack>
             </Infobox>
           )}
-
+          {!!scheduledAt && (
+            <Infobox
+              size="sm"
+              border="1px solid"
+              borderColor="utility.feedback.info"
+              borderRadius="0.25rem"
+            >
+              <Text textStyle="body-2">
+                This page is scheduled for publishing. To make changes, cancel
+                the schedule first.
```

Comment:
nit (quality of life) - should we make this clickable and visually-obvious so that user can click on this to open the modal for better UX?

since this UI is on the left and "cancel schedule" is on the right - might not be super obvious at first glance how to do so